### DISCUSSION TITLE
Is swiftpackageindex.com down?

### DISCUSSION THREAD

**Discussion by macdrevx** [SwiftPackageIndex/SwiftPackageIndex-Server#2848] [Q&A]
Apologies if this isn't the best place to report this, but swiftpackageindex.com has been failing to load for me for the past day or two. I don't see any mention of it on the status page though.

Down for Everyone or Just Me also sees it as down: https://downforeveryoneorjustme.com/swiftpackageindex.com?proto=https

<img width="594" alt="Screenshot 2024-01-18 at 11 08 39 AM" src="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/205658/ded4aa88-fabf-43a6-8d00-3ab4538e7909">

Here's an example of what I'm seeing:

<img width="1039" alt="Screenshot 2024-01-18 at 11 09 00 AM" src="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/205658/26ab4efe-381d-4359-8c24-6a6f7ebd0a6f">

And here's a sample of what I'm seeing in the console:

<img width="845" alt="Screenshot 2024-01-18 at 11 09 44 AM" src="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/205658/25c3cd40-ea3b-4dea-86e9-163597291646">

**Comment by macdrevx** [comment]
the home page just loaded for me, but search doesn't seem to be working

**Comment by macdrevx** [comment]
This banner appeared at the top of the home page after the search failed:

<img width="1253" alt="Screenshot 2024-01-18 at 11 13 51 AM" src="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/205658/b01c704a-4589-42e2-818d-d501f37af739">

**Comment by macdrevx** [comment]
I clicked refresh and now search works & I was able to get to the swift argument parser page, but the documentation isn't loading…

<img width="1269" alt="Screenshot 2024-01-18 at 11 15 57 AM" src="https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/assets/205658/9552a28d-043a-41f1-90f2-c142bb10c3f0">

**Comment by finestructure** [comment]
Hi @macdrevx , thanks for the report! We're having an intermittent issue with one of our machines but best I can tell everything is working as expected at the moment after I made some change just now.

Is everything responsive again for you?

**Reply by finestructure** [reply]
Spoke too soon, still seeing the issue.

**Comment by macdrevx** [comment]
I just went through the following without issue:

1. open home page
2. search for swift-argument-parser
3. click the first result
4. click the documentation link

**Reply by daveverwer** [reply]
Thank you @macdrevx!

We have a old version that deployed about 10 minutes ago. We found an issue in a recent deploy and so we have rolled back a few days.

**Reply by daveverwer** [reply]
We will need some time to investigate the cause of the issue, but hopefully this is the cause.

**Comment by finestructure** [comment]
Ok, I believe we've fixed the issue now. Please report back if you're still seeing timeouts etc.

**Comment by macdrevx** [comment]
Thanks @daveverwer and @finestructure! I'll let you know if I run into any more trouble.
